packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -558,7 +560,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
 
     /// @notice Sets the gas paying token for the L2 system. This token is used as the
     ///         L2 native asset. Only the SystemConfig contract can call this function.
-    function setGasPayingToken(address _token, uint8 _decimals, bytes32 _name, bytes32 _symbol) external {
+    function setGasPayingToken(address _token, uint8 _decimals, bytes32 _name, bytes32 _symbol) external virtual {
```

Comment:
Why is this `virtual`? There is no `override`?

Reply:
the override is in `OptimismPortalInterop` which reimplements the function to use `setConfig`

Reply:
Are you sure that diff exists in the current PR?